test/integration/migrate/index.js

```diff
@@ -268,12 +221,52 @@ module.exports = function(knex) {
 
   });
 
+  describe('knex.migrate.latest in parallel', function() {
+
+    afterEach(function() {
+      return knex.migrate.rollback({directory: 'test/integration/migrate/test'});
+    });
+
+    if (knex.client.dialect === 'postgres') {
+      it("is able to run two migrations in parallel in postgres", function () {
```

Comment:
From what I learned (and tested) auto-commit is enabled by default in MySQL. If the knex user will change this in his database then I expect it will behave the same as Postgres.

Otherwise, it means that one transaction will see that the other transaction has locked and it will cause it to fail (instead of waiting for it to finish). This is consistent with the current behavior of knex migrations.